include/binlog/adapt_struct.hpp

```diff
@@ -154,6 +154,32 @@
   /**/
 #endif // >= C++20
 
+#if __cplusplus >= 202002L // Otherwise __VA_OPT__ breaks
+/**
+ * BINLOG_ADAPT_CONCEPT_NS(Ns, Concept, members...)
+ *
+ * Same as BINLOG_ADAPT_CONCEPT, except for Concepts that are in a namespace.
+ *
+ * Example:
+ *
+ *     namespace ns {
+ *     template<typename T>
+ *     concept Stringable = requires(T a)
+ *     {
+ *       { a.str() } -> std::convertible_to<std::string>;
+ *     };
+ *     }
+ *
+ *     BINLOG_ADAPT_CONCEPT_NS(ns, Stringable, str)
+ *
+ * @see BINLOG_ADAPT_CONCEPT
+ */
+#define BINLOG_ADAPT_CONCEPT_NS(Ns, Concept, ...)                                                                      \
```

Comment:
I would argue that the majority of time concepts will be in namespaces so perhaps the default macro should support that rather than it being the exception? Is it possible for the macro to support passing in the fully qualified type name (including the namespace) similar to how `ADAPT_STRUCT` works or is that not technically feasible?

Reply:
Thanks for your persistence and reminding me how the other parts work already! I changed `BINLOG_ADAPT_CONCEPT` to work with namespaced concepts naturally.